### PR TITLE
hardening/31_logging_facilities

### PR DESCRIPTION
**Pull request by frbattid**
* Implements issue #31 
* No unit tests still available.
* (unofficial) e2e tests passed:
```
$ cat /var/log/cosmos/cosmos-gui.log.2015-07-23 
{"level":"info","message":"Connected to http://130.206.81.225:3306/cosmos_gui","timestamp":"2015-07-23T13:25:20.017Z"}
{"level":"info","message":"cosmos-gui running at http://localhost:9090","timestamp":"2015-07-23T13:25:20.019Z"}
```
* Assignee @gtorodelvalle

### CODE REVIEW

File: cosmos-gui/conf/cosmos-gui.json [hunk 1]
```diff
@@ -30,6 +30,10 @@
     "port": 3306,
     "user": "",
     "password": "",
-    "database": "cosmos"
+    "database": "cosmos
+  },
+  "log": {
+    "file_name": "/var/log/cosmos/cosmos-gui.log",
+    "date_pattern": ".dd-MM-yyyy"
```
Comment (gtorodelvalle) [inline cosmos-gui/conf/cosmos-gui.json]: Not really sure why the previous section appears coloured this way... :O Any clue? :)

File: cosmos-gui/src/logger.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+function debug(message) {
+    winston.debug(message);
+} // debug
+
+module.exports = {
+    info: info,
+    error: error,
+    warn: warn,
+    debug: debug
+} // module.exports
```
Comment (gtorodelvalle) [inline cosmos-gui/src/logger.js]: Blank final line missing... ;)

File: cosmos-gui/conf/cosmos-gui.json (hunk 1 above)
Comment (frbattid) [inline cosmos-gui/conf/cosmos-gui.json]: Curious... I have no idea :)

File: cosmos-gui/src/logger.js (hunk 2 above)
Comment (frbattid) [inline cosmos-gui/src/logger.js]: Fixed in https://github.com/telefonicaid/fiware-cosmos/commit/7bfacc44a935c1cfa6ac471018837d0ba08eda53

**Comment by gtorodelvalle** [comment]
Since we have no idea why the code appears coloured and everything seems to be fine :) , LGTM! Thanks @frbattid !

**Pull request merged**